_posts/02-01-01-Code-Style-Guide.md

```diff
@@ -2,60 +2,43 @@
-applications that implement the components can have consistency even when working with lots of third-party code.
+আদর্শভাবে, কোন একটি প্রচলিত নিয়ম অনুসরণ করে পিএইচপি কোড লিখা উচিৎ। তা হতে পারে অনেকগুলো PSR এর একত্রীকরণ অথবা PEAR বা Zend এর তৈরিকৃত কোন মানদণ্ড। অর্থাৎ অন্যান্য ডেভেলপারেরা যেন সহজেই আপনার কোড পড়তে পারে ও এটি নিয়ে কাজ করতে পারে এবং যেসব অ্যাপ্লিকেশন্স কম্পোনেন্টগুলোকে ইমপ্লিমেন্ট করে তাদের মাঝে যেন সামঞ্জস্যতা থাকে এমনকি যখন অসংখ্য থার্ড-পার্টি কোড নিয়ে কাজ করা হয়।
 
-* [Read about PSR-0][psr0]
-* [Read about PSR-1][psr1]
-* [Read about PSR-2][psr2]
-* [Read about PSR-4][psr4]
-* [Read about PEAR Coding Standards][pear-cs]
-* [Read about Symfony Coding Standards][symfony-cs]
+* [PSR-0 সম্বন্ধে জানুন][psr0]
+* [PSR-1 সম্বন্ধে জানুন][psr1]
+* [PSR-2 সম্বন্ধে জানুন][psr2]
+* [PSR-4 সম্বন্ধে জানুন][psr4]
+* [PEAR কোডিং স্ট্যান্ডার্ডস সম্বন্ধে জানুন][pear-cs]
+* [Symfony কোডিং স্ট্যান্ডার্ডস সম্বন্ধে জানুন][symfony-cs]
 
-You can use [PHP_CodeSniffer][phpcs] to check code against any one of these recommendations, and plugins for text
-editors like [Sublime Text][st-cs] to be given real-time feedback.
+উপরোক্ত নিয়মগুলোর বিপরীতে কোড চেক করার জন্য [PHP_CodeSniffer][phpcs] ব্যবহার করা যেতে পারে এবং [Sublime Text][st-cs] এর মত টেক্সট এডিটর এর প্লাগইনস ব্যবহার করা যেতে পারে যেটি রিয়েল-টাইম ফিডব্যাক প্রদর্শন করে।
 
-You can fix the code layout automatically by using one of the following tools:
+নিচের যেকোন একটি টুলস ব্যবহার করে স্বয়ংক্রিয়ভাবে কোড লেআউট ঠিক করা যায়ঃ
 
-- One is the [PHP Coding Standards Fixer][phpcsfixer] which has a very well tested codebase.
-- Also, the [PHP Code Beautifier and Fixer][phpcbf] tool which is included with PHP_CodeSniffer can be used to adjust your code accordingly.
+- একটি হল [PHP Coding Standards Fixer][phpcsfixer] যার আছে খুব ভালোভাবে পরীক্ষিত কোড ভাণ্ডার।
+- এছাড়াও আছে [PHP Code Beautifier and Fixer][phpcbf] টুল। এই টুলটি PHP_CodeSniffer এ পাওয়া যাবে যার মাধ্যমে কোডকে নিজের পছন্দমত স্টাইল এ সাজানো যায়। .
 
-And you can run phpcs manually from shell:
+এছাড়াও shell থেকে ম্যানুয়ালি নিম্নোক্ত phpcs রান করা যেতে পারেঃ 
 
     phpcs -sw --standard=PSR2 file.php
 
-It will show errors and describe how to fix them.
-It can also be helpful to include this command in a git hook.
-That way, branches which contain violations against the chosen standard cannot enter the repository until those
-violations have been fixed.
+এটি এররগুলো দেখাবে এবং কিভাবে সেগুলো ঠিক করা যায় তা বিস্তারিতভাবে প্রদর্শন করবে। এটি আরও সহায়ক হতে পারে যদি এই কমান্ড একটি git hook এর মধ্যে অন্তর্ভুক্ত করা হয়। এই পদ্ধতিতে, যেসব Branches প্রচলিত নিয়ম লঙ্ঘন করে সেগুলো ঠিক করার আগে রিপোজিটরিতে প্রবেশ করতে পারবে না।
 
-If you have PHP_CodeSniffer, then you can fix the code layout problems reported by it, automatically, with the
-[PHP Code Beautifier and Fixer][phpcbf].
+যদি আপনি PHP_CodeSniffer ব্যবহার করেন, তাহলে এর মাধ্যমে রিপোর্টকৃত কোড লেআউটের সমস্যাগুলো [PHP Code Beautifier and Fixer][phpcbf] এর মাধ্যমে স্বয়ংক্রিয়ভাবে ঠিক করা যায়।
```

Comment:
`যদি আপনি PHP_CodeSniffer ব্যবহার করেন, তাহলে এর রিপোর্টকৃত কোড লেআউটের সমস্যাগুলো [PHP Code Beautifier and Fixer][phpcbf] দিয়ে স্বয়ংক্রিয়ভাবে ঠিক করা যায়।`